pkg/sql/sem/tree/show.go

```diff
@@ -892,3 +892,19 @@ func (n *ShowDefaultPrivileges) Format(ctx *FmtCtx) {
 		ctx.WriteString("FOR ALL ROLES ")
 	}
 }
+
+// ShowCompletions represents a SHOW COMPLETIONS statement.
+type ShowCompletions struct {
+	Statement string
+	Offset    *NumVal
+}
+
+// Format implements the NodeFormatter interface.
+func (s ShowCompletions) Format(ctx *FmtCtx) {
```

Comment:
could use a few test cases in pkg/sql/parser/testdata

Reply:
Done